### PR TITLE
add landing message #1

### PR DESCRIPTION
**Pull request by popdude731**

### CODE REVIEW

**Comment by KeShell** [comment]
Don't know if its ready

**Review by KeShell — COMMENTED** [review]
<h1></h1>

File: index.html [hunk 1]
```diff
@@ -1 +1 @@
-
+<h1>
```
Comment (KeShell) [inline index.html:1]: <h1></h1>

**Review by KeShell — COMMENTED** [review]
Change

File: index.html (hunk 1 above)
Comment (popdude731) [reply index.html:1]: I can change that, yea!

**Review by KeShell — APPROVED** [review]
Approved

File: index.html (hunk 1 above)
Comment (KeShell) [inline index.html:1]: <h1></h1>

**Review by KeShell — CHANGES_REQUESTED** [review]
Should be changed

File: index.html (hunk 1 above)
Comment (KeShell) [reply index.html:1]: Thanks

**Review by KeShell — APPROVED** [review]

**Pull request merged**